corehq/apps/app_manager/templates/app_manager/ng_partials/base_summary_view.html

```diff
@@ -0,0 +1,38 @@
+{% load i18n %}
```

Comment:
I'm sure there are better way to do this but with my still minimal knowledge of Angular this seemed like a good approach at the time.